test/fixtures/tolerant-parse/migrated_0052.js

```diff
@@ -1 +1 @@
-class A {a(eval){}}
\ No newline at end of file
+({x(eval){"use strict"}})
```

Comment:
During @ikarienator 's initial split up, these somehow were reversed. I happened to notice, so figured I'd fix them since I was touching the class one anyway.